.github/workflows/gatk-nightly-docker.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Upload GATK Nightly Docker Image
+on:
+  schedule:
+    # trigger the job the image at 1am est every night (UTC 20:00)
+    # NOTE: by default schedule:cron jobs pull from the repo default branch which makes this the master branch of gatk
+    - cron: '0 20 * * *'
+
+jobs:
+  buildDocker:
+    name: build docker images
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout gatk
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: 'docker.io'
+          username: ${{ secrets.DOCKER_NIGHTLY_SERVICE_LOGIN }}
+          password: ${{ secrets.DOCKER_NIGHTLY_SERVICE_TOKEN }}
+
+      - name: Export tags to env
+        run: |
+          DATE=$(date +%Y-%m-%d);
+          echo "DOCKER_NIGHTLY_TAG=${DATE}-$(git describe)-NIGHTLY-SNAPSHOT" >> $GITHUB_ENV
+          echo "DOCKER_NIGHTLY_TAG_LATEST=latest" >> $GITHUB_ENV
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: broadinstitute/gatk-nightly
+          tags: |
+            type=raw,value=${{env.DOCKER_NIGHTLY_TAG}}
+            type=raw,value=${{env.DOCKER_NIGHTLY_TAG_LATEST}}
```

Comment:
If you feel like learning the [docker tag domain specific language](https://github.com/docker/metadata-action#typeschedule)  you can probably do this same thing with their magic built ins.  I think it's reasonable to NOT do that though.

Reply:
I spent some time looking into that and it looked neat but i figure its probably better to keep the images using the same name scheme as before